### ISSUE TITLE
Support for tax categories and tax jurisdictions

### ISSUE THREAD

**Issue by igniztion** [BVNetwork/CommerceStarterKit#24]
Hard coded 25% tax in several places. Doesn't support e.g. books, that should have 0% or food items 15%. Tax categories not used at all.

**Comment by evest** [comment]
Correct. Do not remember the rationale for this, but I think the original solution did not have the need, so it was never implemented. This affects the payment provider (DIBS), receipt and more.

Shipping tax is also just partially implemented I think, and shipping is not sent to DIBS (needs to be a Line Item there).

When we fully implement the new Promotion system, we have to do a major refactoring of the pricing logic, and this is a good time to look into this. There is no time frame for this however. I will leave this as an open bug until then.
